### ISSUE TITLE
Indents in Python and maintainability

### ISSUE THREAD

**Issue by pjrdata** [matloff/R-vs.-Python-for-Data-Science#19]
Perhaps mention something about the relative advantages and disadvantages of Python's indenting. Simple syntax may be an advantage for educational purposes but for actual production use in the real world, the more structured syntax is far clearer in intent and much easier to maintain in the long term. I have found (and probably caused) numerous errors in production Python code because of stray indents that cause unexpected behaviour. The lack of start/end tokens around code blocks, instead relying on indents for parsing syntax is an abomination in my view :-)

**Comment by matloff** [comment]
Yes, I've mentioned this in the Twitter discussion. Ruby fixes that while remaining Python-like.
